src/Lean/Elab/MutualInductive.lean

```diff
@@ -1006,6 +1181,20 @@ private def elabInductiveViews (vars : Array Expr) (elabs : Array InductiveElabS
       enableRealizationsForConst e.view.declName
     return res
 
+private def elabInductiveViewsCoinductive (vars : Array Expr) (elabs : Array InductiveElabStep1) : TermElabM Unit := do
```

Comment:
If I read the code below correctly, this is **not** about elaborting the coinductive, but rather the flat inductive, is it?
```suggestion
private def elabFlatInductiveViews (vars : Array Expr) (elabs : Array InductiveElabStep1) : TermElabM Unit := do
```